src/MovieForm.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from 'react';
+
+export default function MovieForm({ submitMovie, onMovieChange }) {
+  const [movie, setMovie] = useState({});
```

Comment:
I love this idea of holding a movie object in state and just updating each key as the user inputs it. well done!